diff --git a/node_modules/@teritorio/openstreetmap-logical-history-component/dist/index.js b/node_modules/@teritorio/openstreetmap-logical-history-component/dist/index.js
index ab33954..03a9485 100644
--- a/node_modules/@teritorio/openstreetmap-logical-history-component/dist/index.js
+++ b/node_modules/@teritorio/openstreetmap-logical-history-component/dist/index.js
@@ -2208,7 +2208,7 @@ var Hr = ["id"], Ur = /* @__PURE__ */ Ct(/* @__PURE__ */ l({
 		ee((e) => ({ v223c9cda: f.value }));
 		let l = c, u = te(), d = t(() => !!u["content-start"]), f = t(() => d.value ? "repeat(4, minmax(0, 1fr))" : "repeat(3, minmax(0, 1fr))"), m = t(() => d.value ? 3 : 2), h = t(() => d.value ? 4 : 3), _ = p(tt), { loCha: y, getBeforeFeatures: b, getAfterFeatures: x } = p(et);
 		if (!y.value) throw Error("LoCha is empty.");
-		let C = t(() => b(l.features)), w = t(() => x(l.features)), T = t(() => C.value.length > 0 && w.value.length === 0), O = t(() => C.value.length === 0 && w.value.length > 0), A = t(() => C.value.length === 1 && w.value.length === 1 && !!w.value[0]?.properties.deleted), j = t(() => C.value.length === 1 && w.value.length === 1 && !w.value[0]?.properties.deleted), ne = t(() => {
+		let C = t(() => b(l.features)), w = t(() => x(l.features)), T = t(() => C.value.length > 0 && w.value.length === 0), O = t(() => C.value.length === 0 && w.value.length > 0), A = t(() => C.value.length === 1 && w.value.length === 1 && !!w.value[0]?.properties.deleted), j = t(() => !y.value?.metadata.forceMultiColumn && C.value.length === 1 && w.value.length === 1 && !w.value[0]?.properties.deleted), ne = t(() => {
 			let e = y.value?.metadata.links[l.index] ?? [];
 			return new Map(w.value.map((t) => {
 				let n = e.filter((e) => e.after === t.id).map((e) => e.before).filter(Boolean);
